docs/install-pro.md

```diff
@@ -0,0 +1,97 @@
+# Install Percona XtraBackup Pro
+
+--8<--- "pro-build-announcement.md"
+
+This document provides guidelines how to install Pro packages of Percona XtraBackup from Percona repositories. [Check files in packages built for Percona XtraBackup Pro :material-arrow-right:](pro-files.md){.md-button}
+
+## Procedure
+
+1. Request the access to the pro repository from Percona Support. You will receive the client ID and the access token which you use when downloading the packages.
+
+3. Configure the repository and install Percona XtraBackup packages
+
+    === "On Debian or Ubuntu"
+
+        1. Download a `DEB` package for *percona-release* the repository packages from Percona web:
+
+            ```{.bash data-prompt="$"}
+            $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb
+            ```
+
+        2. Install the downloaded package with **dpkg**. To do that, run the following commands as root or with **sudo**: `dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb`
+   
+            Once you install this package the Percona repositories should be added. You
+            can check the repository setup in the `/etc/apt/sources.list.d/percona-release.list` file.
+
+        3. Enable the repository: `percona-release enable-only tools release`
```

Comment:
We do not release PXB 8.4.x packages in "tools" release repository anymore

Reply:
Created this ticket to fix already published 8.4 documentation: 
https://perconadev.atlassian.net/browse/PXB-3424

Page to be fixed: https://docs.percona.com/percona-xtrabackup/8.4/apt-repo.html

Reply:
updated